src/styles/_variables.scss

```
@@ -0,0 +1,5 @@
+$cell-size: 100px;
+$col-count: 7;
+$padding: 10px;
+$days: "mon" 0, "tue" 1, "wed" 2, "thu" 3, "fri" 4, "sat" 5, "sun" 6;
```

```suggestion
$days: 
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
```

Just for better readability